core_backend/app/routers/manage_content.py

```diff
@@ -87,9 +113,29 @@ async def retrieve_content(
     return contents
 
 
-@router.delete("/{content_id}/delete")
+@router.get("/summary", response_model=list[ContentSummary])
+async def retrieve_content_summary(
+    readonly_access_user: Annotated[
+        AuthenticatedUser, Depends(get_current_readonly_user)
+    ],
+    language: int,
```

Comment:
Should we use the front-end to apply language filters? @Tanmay-97 @amiraliemami  what do you think?

Reply:
I am resolving this for now!